modules/github-oidc-role/README.md

```diff
@@ -0,0 +1,190 @@
+# Component: `github-oidc-role`
+
+This component is responsible for creating IAM roles for GitHub Actions to assume.
+
+## Usage
+
+**Stack Level**: Regional
+
+Here's an example snippet for how to use this component.
+
+```yaml
+# stacks/catalog/gha-service-roles/defaults.yaml
```

Comment:
replace `gha-service-roles` with `github-oidc-role`. plus other references in this README